client/my-sites/domains/domain-management/transfer/transfer-out/locked.jsx

```diff
@@ -58,65 +59,115 @@ const Locked = React.createClass( {
+				{ translate(
 					'This Top Level Domain (TLD) requires that we manually request a ' +
 					'transfer code on your behalf. After we have received it, we will ' +
 					'email it to you.'
 				) }
 			</p>
 		);
-	},
+	}
 
-	handleGiveMeTheCodeClick( event ) {
+	handleGiveMeTheCodeClick = ( event ) => {
 		event.preventDefault();
 		this.requestTransferCode();
-	},
+	}
+
+	renderFooter = () => {
+		const { translate } = this.props;
+
+		if ( this.state.showFooter ) {
+			return (
+				<div>
+					<p className="transfer-out__small-text">
+						<a href="" onClick={ this.handleGiveMeTheCodeClick }>{ translate( 'I just want the transfer code for now.' ) }</a>
+					</p>
+					{ this.isManualTransferRequired() && this.renderManualTransferInfo() }
+					<Button
+						className="transfer-out__action-button"
+						onClick={ this.handleTransferClick }
+						primary
+						disabled={ this.state.submitting }>
+						{ translate( 'Update Settings And Continue' ) }
+					</Button>
+				</div>
+			);
+		}
+	}
+
+	handleEnableTransferButtonClick = () => {
+		this.setState( { transferButtonDisabled: false } );
+	}
 
 	render() {
+		const { translate } = this.props;
 		const { privateDomain } = getSelectedDomain( this.props );
 		return (
 			<div>
-				<SectionHeader label={ this.translate( 'Transfer Domain' ) }/>
+				<SectionHeader label={ translate( 'Transfer Domain' ) } />
 				<Card className="transfer-card">
 					<div>
 						<p>
 							{ privateDomain
-								? this.translate( 'To transfer your domain, we must unlock it and remove Privacy Protection. ' +
+								? translate( 'To transfer your domain, we must unlock it and remove Privacy Protection. ' +
 									'Your contact information will be publicly available during the transfer period.' )
-								: this.translate( 'To transfer your domain, we must unlock it.' )
+								: translate( 'To transfer your domain, we must unlock it.' )
 							} <a
 									href={ support.TRANSFER_DOMAIN_REGISTRATION }
-									target="_blank" rel="noopener noreferrer">{ this.translate( 'Learn More.' ) }</a>
+									target="_blank" rel="noopener noreferrer">{ translate( 'Learn More.' ) }</a>
 						</p>
-						<p className="transfer__small-text">
-							<a href="" onClick={ this.handleGiveMeTheCodeClick }>{ this.translate( 'I just want the transfer code for now.' ) }</a>
+						<div>
+							<Notice
+								showDismiss={ false }>
+								<div>
+									The contact informaion for this domain was changed <strong>24 days ago</strong>. Under ICANN policy,
+									these changes trigger a 60-day transfer lock to be applied to the domain unless you opt out during
+									the process. <a>Learn more.</a>
+								</div>
+									<Button
+										className="transfer-out__show-footer"
+										onClick={ this.handleEnableTransferButtonClick }
+										primary
+										compact
+										disabled={ ! this.state.transferButtonDisabled }>
+										{ translate( 'I Understand' ) }
```

Comment:
Hi! I've found a possible matching string that has already been translated **19** times:
`translate( 'I understand that:' )` *ES Score: 16*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).